.changeset/wet-crabs-guess.md

```diff
@@ -0,0 +1,6 @@
+---
+'@backstage/backend-plugin-api': patch
+'@backstage/plugin-catalog-backend': patch
+---
+
+Start using the `isDatabaseConflictError` helper from the `backend-plugin-api` package in order to avoid dependency with the soon to deprecate `backend-common` package.
```

Comment:
```suggestion
Start using the `isDatabaseConflictError` helper from the `@backstage/backend-plugin-api` package in order to avoid dependency with the soon to deprecate `@backstage/backend-common` package.
```

Although tbh I think this message could simply be "Updated deprecated imports" or something like that, just to not clutter the changelog too much